pkg/services/github.go

```diff
@@ -326,7 +324,29 @@ func (g gitHubService) Send(notification Notification, _ Destination) error {
 	if len(u) < 2 {
 		return fmt.Errorf("GitHub.repoURL (%s) does not have a `/`", notification.GitHub.repoURL)
 	}
-	if notification.GitHub.Status != nil {
+
+	var (
+		sendStatus     bool
+		sendDeployment bool
+		sendComment    bool
+	)
+
+	switch dest.Recipient {
```

Comment:
i would suggest instead create variables, split them into different functions and execute inside switch

Reply:
Updated. Couldn't use switch without making it very complex.